ArcTouchChallenge/ArcTouchChallenge/Model/Movie/Genres.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  Genres.swift
+//
+//  Created by Arthur Augusto Sousa Marques on 3/14/18
+//  Copyright (c) . All rights reserved.
+//
+
+import Foundation
+import SwiftyJSON
+
+class Genres: Model {
+
+  // MARK: Declaration for string constants to be used to decode and also serialize.
+  private struct SerializationKeys {
+    static let id = "id"
+    static let name = "name"
+  }
+
+  // MARK: Properties
+  public var id: Int?
+  public var name: String?
+
+  // MARK: SwiftyJSON Initializers
+  /// Initiates the instance based on the object.
+  ///
+  /// - parameter object: The object of either Dictionary or Array kind that was passed.
+  /// - returns: An initialized instance of the class.
+  public convenience init(object: Any) {
```

Comment:
This was created automatically from the component third party and wasn't necessary yet to be used, but it could be if there was a necessity to create a object hard coded.